elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/QueryDsl.scala

```diff
@@ -35,7 +35,7 @@ trait QueryDsl extends DslCommons with SortDsl {
   case class QueryRoot(query: Query,
                        fromOpt: Option[Int],
                        sizeOpt: Option[Int],
-                       sort: Seq[Sort],
+                       sortOpt: Option[Seq[Sort]],
```

Comment:
Here is the option if you want it to stay the same:
`(if(sort.isEmpty) None else Some(sort)).map(_sort -> _.map(_.toJson))`
I feel it is better suited as an Option though since all the other fields are Options (including sourceFilter which is also a Seq).  Let me know your thoughts, will change to your opinion.